Makefile

```diff
@@ -18,20 +18,18 @@ generate:
 	cp plugins.json data/static/
 	go generate ./...
 
-## Runs govet and gofmt against all packages.
+## Runs go vet and golangci-lint against all packages.
 .PHONY: check-style
-check-style: govet lint
-
-## Runs govet against all packages.
-.PHONY: govet
-govet:
+check-style:
 	go vet ./...
```

Comment:
Do we still need this?

Reply:
With it, I get a nice clean error message on a syntax error:
```
> make check-style
go vet ./...
# github.com/mattermost/mattermost-marketplace/cmd/marketplace
vet: cmd/marketplace/main.go:18:30: missing ',' before newline in argument list (and 6 more errors)
make: *** [check-style] Error 2
```

without it ... not so much:
```
make check-style
golangci-lint run ./...
WARN [runner] Can't run linter goanalysis_metalinter: SA5007: failed prerequisites: buildssa@github.com/mattermost/mattermost-marketplace/cmd/marketplace
WARN [runner] Can't run linter unused: buildssa: analysis skipped: errors in package: [/Users/jesse/Documents/mattermost/src/marketplace/cmd/marketplace/main.go:18:30: missing ',' before newline in argument list /Users/jesse/Documents/mattermost/src/marketplace/cmd/marketplace/main.go:19:1: expected operand, found '}' /Users/jesse/Documents/mattermost/src/marketplace/cmd/marketplace/main.go:22:2: missing ',' in argument list /Users/jesse/Documents/mattermost/src/marketplace/cmd/marketplace/main.go:23:48: missing ',' before newline in composite literal /Users/jesse/Documents/mattermost/src/marketplace/cmd/marketplace/main.go:24:13: missing ',' before newline in composite literal /Users/jesse/Documents/mattermost/src/marketplace/cmd/marketplace/main.go:25:3: missing ',' before newline in argument list /Users/jesse/Documents/mattermost/src/marketplace/cmd/marketplace/main.go:26:1: expected operand, found '}' /Users/jesse/Documents/mattermost/src/marketplace/cmd/marketplace/main.go:18:30: missing ',' before newline in argument list /Users/jesse/Documents/mattermost/src/marketplace/cmd/marketplace/main.go:19:1: expected operand, found '}' /Users/jesse/Documents/mattermost/src/marketplace/cmd/marketplace/main.go:22:2: missing ',' in argument list /Users/jesse/Documents/mattermost/src/marketplace/cmd/marketplace/main.go:23:48: missing ',' before newline in composite literal /Users/jesse/Documents/mattermost/src/marketplace/cmd/marketplace/main.go:24:13: missing ',' before newline in composite literal /Users/jesse/Documents/mattermost/src/marketplace/cmd/marketplace/main.go:25:3: missing ',' before newline in argument list /Users/jesse/Documents/mattermost/src/marketplace/cmd/marketplace/main.go:26:1: expected operand, found '}']
```